src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataInfo.cs

```diff
@@ -32,6 +34,18 @@ public ViewDataInfo(object container, PropertyInfo propertyInfo, Func<object> va
             _valueAccessor = valueAccessor;
```

Comment:
We don't have a goal of obsoleting APIs just because we don't use them. We obsolete APIs only when they become obsolete...